### ISSUE TITLE
tutorial text should tell about beacons

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#2208]
Currently, there is no way to discover about the beacon entries, except by going through the bindings.
I think the tutorial could mention this feature.

This means more text in there, which is not nice, but it makes a lot more sense to me than telling people how to use the console, for example.

**Comment by necessarily-equal** [comment]
(for the record, the tutorial here is the text on the left that explains key bindings)

**Comment by slipher** [comment]
I don't agree. The tutorial text is not meant to be a complete manual. If it includes less important details, that makes it less useful for starting to the play the game because it is hard to find the important ones. (If opening the console is in there, it would be good to remove that too)

**Comment by bmorel** [comment]
> If it includes less important details, that makes it less useful for starting to the play the game  [...] (If opening the console is in there, it would be good to remove that too)

I agree on that. Information should be reduced to minimum, and I think the beacon menu is much more important than the console in the v1 (still have hope).
So, in all cases the console info should be removed, and I think the beacon info would be nice to have, but on this one you disagree (and it's fine).
